src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -227,6 +228,25 @@ private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string
             }
         }
 
+        private static void AddAcceptsRequestFormats(
+            IList<ApiRequestFormat> supportedRequestFormats,
+            EndpointMetadataCollection endpointMetadata)
+        {
+            var requestMetadata = endpointMetadata.GetOrderedMetadata<IAcceptsMetadata>();
```

Comment:
How does the matcher policy work if there's more than one AcceptsMetadata i.e. if I do

```
MapPost("...", () => {}).Accepts("application/json").Accepts("application/xml")
```

Does it allow json?

Reply:
from my preliminary test, yes, it allows both.

Reply:
That seems strange given the policy is looking up exactly one item - https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Core/src/Routing/ConsumesMatcherPolicy.cs#L78-L79?